portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,28 +30,19 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     private static final String content = "content";
```

Comment:
I notice this is in all caps in the other PR but not here. Checking the git log, it looks like your branches have diverged significantly. Can you merge/rebase this PR on top of the other one? It looks like there's some conflicts.

Reply:
Code compiles now :)

For these `private static final` variables, can you ensure the variable names are in UPPER_SNAKE_CASE like `COMMENT_PARAMETER`?